lnbits/core/models/misc.py

```diff
@@ -38,6 +38,11 @@ class SimpleStatus(BaseModel):
     message: str
 
 
+class SimpleItem(BaseModel):
```

Comment:
its very generic name and hard to know where it belongs, can we maybe do something like `AclTokenListItem`

Reply:
Its generic indeed and located in `models/misc.py`.
It can be used in different context.